packages/credential-provider-process/src/getValidatedProcessCredentials.spec.ts

```diff
@@ -18,7 +18,7 @@ describe(getValidatedProcessCredentials.name, () => {
     Expiration: mockExpiration,
   });
 
-  it.each([undefined, 2])("throws Error when Version is %s", (Version) => {
+  it.each([2])("throws Error when Version is %s", (Version) => {
```

Comment:
is this lost case not useful?

Reply:
The case is not supported as per TypeScript type definitions.